src/dotnet-compile-fsc/project.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.0.0-*",
+    "compilationOptions": {
+        "emitEntryPoint": true
+    },
+
+    "dependencies": {
+        "Microsoft.FSharp.Compiler.netcore": "1.0.0-alpha-160318",
+        "System.CommandLine": "0.1.0-e160323-1",
+
+        "Microsoft.DotNet.ProjectModel": "1.0.0-*",
```

Comment:
I don't think so, dotnet-compile-fsc is going to be shipped from another repo very soon, tried to replicate that scenario as much as possible.